nimble/host/mesh/src/transport.c

```diff
@@ -590,8 +590,9 @@ static int sdu_recv(struct bt_mesh_net_rx *rx, u32_t seq, u8_t hdr,
 
 	if (IS_ENABLED(CONFIG_BT_MESH_FRIEND) && !rx->local_match) {
 		BT_DBG("Ignoring PDU for LPN 0x%04x of this Friend",
-		       rx->ctx.recv_dst);
-		return 0;
+			rx->ctx.recv_dst);
```

Comment:
This part still needs whitespace fix:

```suggestion
		       rx->ctx.recv_dst);
```

Reply:
In other words, mesh code was ported from Zephyr and the coding style should be the same. So you should use tabs and then align to the beginning of parenthesis with spaces.